rally-domain/src/main/groovy/yakworks/rally/orgs/repo/OrgSourceRepo.groovy

```diff
@@ -93,8 +93,13 @@ class OrgSourceRepo implements GormRepo<OrgSource>, IdGeneratorRepo {
 
     }
 
-    OrgSource findBySourceIdAndOrgType(String sourceId, OrgType orgType) {
-        return OrgSource.find("from OrgSource where sourceId = :sourceId and orgType = :orgType ",
-            [sourceId: sourceId, orgType: orgType])
+    OrgSource findBySourceIdAndOrgType(String theSourceId, OrgType theOrgType) {
+        // return OrgSource.find("from OrgSource where sourceId = :sourceId and orgType = :orgType ",
+        //     [sourceId: sourceId, orgType: orgType])
+
+        return OrgSource.where { sourceId == theSourceId && orgType == theOrgType }.get()
```

Comment:
so are we using "where" even that it's not the fastest way? Should we add XXX to make it work with find ?